packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -236,10 +240,19 @@ export function getAllTaggedElements(root = window.document, ignoreManual?: bool
     element.setAttribute(InspectorModeDataAttributes.LOCALE, sourceMap.contentful.locale);
     element.setAttribute(InspectorModeDataAttributes.SPACE, sourceMap.contentful.space);
     element.setAttribute(InspectorModeDataAttributes.ENVIRONMENT, sourceMap.contentful.environment);
+    element.setAttribute(InspectorModeDataAttributes.AUTO_TAGGED, 'true');
     taggedElements.push(element);
   }
 
-  return taggedElements;
+  const autoTaggedCount = Array.from(manualTagged).filter((el) =>
+    el.hasAttribute(InspectorModeDataAttributes.AUTO_TAGGED),
```

Comment:
instead of auto-tagged could we add a property for manual tagging (similar to vercel is doing it for their visual editing)

Caveat: We would need to check for the SDK version (before the current version it was always manual tagged)

Reply:
Yup I'll adjust this now